frontend/src/components/home/SearchBar.jsx

```diff
@@ -2,15 +2,46 @@ import React from "react";
 import ReactDOM from "react-dom";
 import { useState } from "react";
 import "./SearchBar.css";
+import { getAllBoards } from "../../utils/reusedFunctions";
```

Comment:
Looks like this import is overwritten in the SearchBar props and not otherwise used? I think you could remove it from utils.